app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -0,0 +1,166 @@
+
+    options = @defaultChartOptions
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData);
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-fixed-cost-chart'));
+    chart.draw(chartData, options);
+
+  breakevenProductProfitChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.product'),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.profit'),"pattern":"","type":"number"}
+    ]
+
+    rows = []
+    for key,product of @breakevenProductProfit()
+      name = key
+      total = product.profit
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    options = @defaultChartOptions
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData);
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-product-profit'));
+    chart.draw(chartData, options);
+
+  breakevenProductionCostsChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.product'),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.production_cost'),"pattern":"","type":"number"}
+    ]
+
+    options = @defaultChartOptions
+    rows = []
+    for key,product of @breakevenProductionCosts
+      name = product.name
+      total = product.total
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData);
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-production-cost-chart'));
+    chart.draw(chartData, options);
+
+  breakevenProductProfit: () ->
+    profitData = {}
+
+    for key,product of @breakevenRevenue
+      name = product.name
+      revenue = product.total
+
+      profitData[name] = {revenue: revenue}
+
+    for key,product of @breakevenProductionCosts
+      name = product.name
+      cost = product.total
+
+      profitData[name]['cost'] = cost
+      profitData[name]['profit'] = profitData[name]['revenue'] - cost
+
+    return profitData
+
+  breakevenRevenueChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.product'),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.revenue'),"pattern":"","type":"number"}
+    ]
+
+    options = @defaultChartOptions
+    rows = []
+    for key,product of @breakevenRevenue
+      name = product.name
+      total = product.total
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData);
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-revenue-chart'));
+    chart.draw(chartData, options);
+
+  breakevenTotalCostsChart: () ->
```

Comment:
I did not have time to change this chart over to the desired chart that shows total costs only rather than the fixed vs. product breakdown. The colors have been changed to match the hue of the default Google Charts colors. Those two base colors (blue and red) match the Google Charts hue.